clusterloader2/go.mod

```diff
@@ -55,6 +55,7 @@ require (
 	k8s.io/client-go v0.22.15
 	k8s.io/component-base v0.22.15
 	k8s.io/component-helpers v0.22.15
+	k8s.io/klog v1.0.0 // indirect
```

Comment:
let's try to find out who is pulling this and fix it there, we should only have `k8s.io/klog/v2`